common/generate_bundle_base

```diff
@@ -23,7 +23,9 @@ if [[ -z $vip_start ]] && [[ -e ~/novarc ]]; then
         # Our primary network is /25 and we are arbitrarily using the
         # last 20 addresses for vips which is prone to collisions but
         # we have no alternative currently.
-        vip_start=$(echo $cidr| sed -r 's/([0-9]+\.[0-9]+\.[0-9]+).+/\1/g').106
+        net_start=$(ipcalc -b $cidr |grep HostMin | awk '{print $2}')
```

Comment:
You can avoid the extra pipe with: `net_start=$(ipcalc -b $cidr | awk '/HostMin/{print $2}')`

Alternatively, can also simplify these lines with (which avoids pipes altogether):
`net_start_start=$(awk -F'.' '/HostMin/{print $NF}' <<<$(ipcalc -b $cidr))`

Reply:
nice, I like that, will amend